src/ortc.cpp

```diff
@@ -336,7 +337,8 @@ namespace mediasoupclient
 				MSC_THROW_TYPE_ERROR("missing codec.mimeType");
 
 			std::smatch mimeTypeMatch;
-			std::regex_match(mimeTypeIt->get<std::string>(), mimeTypeMatch, MimeTypeRegex);
+			std::string tmp = mimeTypeIt->get<std::string>();
+			std::regex_match(tmp, mimeTypeMatch, MimeTypeRegex);
```

Comment:
Same as before.

Reply:
There was a change going from C++11 to C++14 where std::regex_search is no longer allowed to take a r-value